.kitchen.yml

```diff
@@ -90,6 +94,7 @@ suites:
         cfn_cluster_user: <%= ENV['CFN_CLUSTER_USER'] %>
         cfn_sqs_queue: <%= ENV['CFN_SQS_QUEUE'] %>
         cfn_ddb_table: <%= ENV['CFN_DDB_TABLE'] %>
+        cfn_base_os: <%= ENV['CFN_BASE_OS'] %>
         custom_node_package: <%= ENV['PARALLELCLUSTER_NODE_URL'] %>
         os: <%= ENV['OS'] %>
```

Comment:
Minor: we are also exporting a kitchen test OS here. On a first glance it seems a bit strange that we are exporting OS in 2 different places. When looking in the code, `ENV['OS']` is used to separate out different kitchen test cases, such as `centos7` and `centos7-custom`.
To make things more obvious, I would maybe rename this to `kitchen_test_os: <%= ENV['OS'] %>`? And/or add a line of comment showing the example of `centos7` vs `centos7-custom`
If we want to change this let's change it everywhere in this file.

Reply:
I added some comments. Changing the name requires code change in other files. I would prefer keep the current name.

Reply:
Sounds good.

On another note, comments can be a lot shorter, we want to describe the attributes rather than listing all available values.
For `cfn_base_os` we can simply add a comment saying something like, "base_os supported by pcluster"
For `os` we can simply say something like, "os attribute used in kitchen test pipeline, i.e. centos7/centos7-custom"